docs/source/ops/common-metrics.rst

```diff
@@ -247,3 +247,40 @@ daml_services_pruning_prune_completed_total
 - **Description**: Total number of completed pruning processes.
 - **Type**: Counter
 
+JVM Metrics
+************
+We expose the following metrics for the JVM if enabled.
+
+runtime_jvm_gc_time
+==============================
+- **Description**: Time spent in a given JVM garbage collector in milliseconds.
+- **Labels**:
+
+  - **gc**: Garbage collector
```

Comment:
Can you link the possible values here, please? I'm assuming this is `Serial`, `Parallel`, `G1`, `Z`, etc.